src/AppBundle/Repository/PostRepository.php

```diff
@@ -28,6 +28,14 @@
 class PostRepository extends EntityRepository
 {
     /**
+     * @return Post[]
+     */
+    public function findAll()
```

Comment:
Overriding the builtin doctrine methods to change their behavior is a bad practice IMO, so you should use a different name